config/go.d/prometheus.conf

```diff
@@ -207,6 +207,9 @@ jobs:
   # https://github.com/prometheus/prometheus/wiki/Default-port-allocations
   #  - name: node_exporter_local
   #    url: 'http://127.0.0.1:9100/metrics'
+  # example to pull from node exporter demo
```

Comment:
and you need that why? Can you elaborate in the comment

Reply:
added comment to pr desc. 

mainly its to give them a really opbvious of a "real" prometheus endpoint out there in the wild that they can just point at. 

i found all the existing ones sort of assume or suggest you need to install prometheus locally and then can scrape `http://127.0.0.1:9100/metrics` for example. 

this is just to try make it even more obvious that all you need is a url with prom metrics on it by showing a "real" (demo) node exporter. 

had a look at the demo site does look to be a proper one created by the promethus team: https://demo.do.prometheus.io/
https://github.com/prometheus/demo-site

Reply:
I mean add reasoning to the comment

> example to pull from node exporter demo

Reply:
Oh sorry, yep totally will do

Reply:
> i found all the existing ones sort of assume or suggest you need to install prometheus locally and then can scrape http://127.0.0.1:9100/metrics for example.

Yes, that is the assumption we use everywhere - we do try to discover stuff that runs locally.

Reply:
> this is just to try make it even more obvious that all you need is a url with prom metrics on it by showing a "real" (demo) node exporter.

If the goal of this PR is to highlight that you can use a remote endpoint and not localhost only then I politely disagree. I doubt such confusion is possible.

We have local/remote examples in the [readme](https://github.com/netdata/go.d.plugin/tree/master/modules/prometheus#configuration). Perhaps you can update it.

Reply:
I mean, uncomment some lines in a conf file and then just "see" it working.

Reply:
But yeah perhaps just mentioning the example I have here in the readme is enough

Reply:
> uncomment some lines in a conf file and then just "see" it working.

That makes sense. On the other hand - if a person made it to the config file it is very likely that they not just wondering around but have an endpoint to scrape.

Reply:
>  I have here in the readme is enough

lgtm, lets do this way